ugs-core/test/com/willwinder/universalgcodesender/BufferedCommunicatorTest.java

```diff
@@ -594,10 +594,5 @@ public boolean processedCommand(String response) {
         public boolean processedCommandIsError(String response) {
             return (response != null && response.startsWith("error"));
         }
-
-        @Override
-        public String getLineTerminator() {
-            return "\r\n";
```

Comment:
Where do these line terminator constants live now?

Reply:
This method was never used and I was a bit surprised that it's hard coded in the BufferedCommunicator. If there is need for such a method I think it should exist in the Controller as the line terminator is protocol-specific?

Reply:
Interesting, it looks like this was deprecated last year by the `ResponseMessageHandler`. It is hard coded in subclasses of BufferedCommunicator, along with some other simple extensions to the communicator.

Things like pausing the stream if there is an error don't work as well if it has to wait for the events to be sent all the way back to the controller, there were other weird cases where the controller would be paused but the controller wasn't (and vice-versa). Probably fine to leave it this way, but it does make the "Communicator" abstraction less useful. i.e. an alternative `RxCommunicator` couldn't be dropped in.